File: Fibonacci/fib.java
class fib{
	static int fib(int n){
		if(n==0|n==1){
			return 1;
		}
		return fib(n-1)+fib(n-2);
	}
	public static void main(String[] args){
		int n = 40;
		System.out.printf("fib(%d) = %d\n", n, fib(n));
	}
}

File: Fibonacci/fib.vb
Module Module1

    Sub Main()
        Dim n As Integer = 40
        System.Console.WriteLine("Fib(" & n & ") = " & Fib(n))

    End Sub
    Function Fib(ByVal n As Integer) As Integer
        If n = 0 OrElse n = 1 Then
            Return 1
        Else
            Return Fib(n - 1) + Fib(n - 2)
        End If
    End Function

End Module
